app/Repositories/ReportbackRepository.php

```diff
@@ -179,11 +181,47 @@ public function updateReportbackItems($data)
         foreach ($data as $reportbackItem) {
```

Comment:
@DFurnes in the new interface, you would just review one at a time, but in Phoenix you review a bunch and then submit. This is one of the reasons why I brought up what I did in Slack. I would like to not have to support the Phoenix interface at all in Rogue.